Dart/src/com/jetbrains/lang/dart/ide/runner/server/vmService/DartVmServiceListener.java

```diff
@@ -78,6 +78,11 @@ public void received(@NotNull final String streamId, @NotNull final Event event)
           onIsolatePaused(event.getIsolate(), breakpoints, exception, event.getTopFrame(), event.getAtAsyncSuspension());
         });
         break;
+      case PausePostRequest:
+        // We get this event after an isolate reload call, when pause after reload has been requested.
+        // TODO: Set current breakpoints; resume the isolate.
+        myDebugProcess.resumeIsolate(event.getIsolate().getId(), null);
```

Comment:
I'll replace with `myDebugProcess.getVmServiceWrapper().resumeIsolate(isolateRef.getId(), null)` to avoid introducing a delegating method in `DartVmServiceDebugProcess` class.